.travis.yml

```diff
@@ -1,17 +1,10 @@
-# trusty beta image has jdk8, gcc4.8.4
-dist: trusty
 sudo: required
-# xcode8 has jdk8
-osx_image: xcode8
-# Not technically required but suppresses 'Ruby' in Job status message.
-language: java
 
 os:
   - linux
-  - osx
```

Comment:
no more osx? I don't have the context.

Reply:
Are these statements in the commit message not enough?

```
This change also does the following:
[...]
* Bump Bazel version to 0.29.1 in .travis.yml file
[...]
Travis changes:
    
    * Remove dist: trusty (Ubuntu 14.04 is EOL, default is 16.04)
    * Remove osx OS as it currently doesn't support Python: [1].

[1] https://github.com/travis-ci/travis-ci/issues/2312
```

Reply:
Yeah, it's verified by Buildkite (and Travis is super slow), so there's no harm in removing it.

See also https://github.com/bazelbuild/rules_closure/pull/422#issuecomment-531787668